_quarto.yml

```diff
@@ -40,6 +26,17 @@ format:
     df-print: paged
     fig-width: 8
     fig-height: 6
+    title: "Scaling Up With R and Arrow"
+    subtitle: "Bigger Data, Easier Workflows"
+    author: "Nic Crane, Jonathan Keane, and Neal Richardson"
+    reader-mode: true
+    google-analytics: "G-4JT42QR6DV"
+    page-footer:
+      left: |
+        Scaling Up With R and Arrow was written by Nic Crane, Jonathan Keane,
+        and Neal Richardson.
+      right: |
+        This book was built with <a href="https://quarto.org/">Quarto</a>.
   pdf:
     toc: false
```

Comment:
So, to be clear, you solved this by adding `before_body.text` and including `\tableofcontents` in that?

Reply:
Yeah, I know there's a quarto setting to do it but in the other examples I saw they do it this way

Reply:
Oh wait, I have 2 frontmatter tags...